apps/frontend/src/utils/constants.ts

```diff
@@ -7,7 +7,7 @@ export const sovrynLinks = {
   telegram: 'https://t.me/SovrynBitcoin',
   twitter: ' https://twitter.com/SovrynBTC',
   github: 'https://github.com/DistributedCollective',
-  blog: 'https://www.sovryn.app/blog',
+  blog: 'https://sovryn.app/all-things-sovryn',
```

Comment:
```suggestion
  blog: 'https://sovryn.com/all-things-sovryn',
```
I believe this should be `sovryn.com` and not `sovryn.app`, am I right @soulBit ?

Reply:
on the 9th, we'll deploy D2 to sovryn.app

Reply:
I know but `.app` should be used only for apps and `.com` is where our blog is.

Reply:
@pietro-maximoff D2 will live on `sovryn.app` and `test.sovryn.app`, tiltom is correct here